Library/Homebrew/.rubocop.yml

```diff
@@ -1,3 +1,5 @@
+require: rubocop-rspec
```

Comment:
I can test and see if that works since I think that would be the best solution.

Reply:
Nevermind that comment =). I think that the problem is that `rspec` is referenced in the `.rubocop_todo.yml` file which doesn't inherit `test/.rubocop.yml` . We could require it from `Homebrew/.rubocop.yml` but it wouldn't really make a difference since it would accomplish the same thing. Another suggestion might be to create the `.rubocop_todo.yml` in the `test` directory.

Reply:
@javian Yes, the test directory could have it's own `.rubocop_todo.yml` which is `require`d by the `.rubocop.yml` in the same directory.

Reply:
If I try to move it to the test directory I currently get this
```
An error occurred while RSpec/PredicateMatcher cop was inspecting /usr/local/Homebrew/Library/Homebrew/software_spec.rb:409:4.
```
that seems to trigger for the *_spec.rb files. I'll see if I can figure it out.

Reply:
```
➜  Homebrew git:(moverubocopoption) ✗ find . -name *_spec.rb
./extend/os/software_spec.rb
./extend/os/linux/software_spec.rb
./software_spec.rb
```

Reply:
I've pushed what I got now. It seems to work but it excludes the *_spec.rb files (except in `test/`) since `rubocop-rspec` checks that pattern by default.

Reply:
We should consider renaming those *_spec.rb files I think since they are not rspec specs and the problem should go away =)